*.Tests.ps1 files with no Describe blocks get executed

Hi,

Just tried to run your extension. It doesn't go well.
1. Once I start project I can see this icon going in circles. It never stops

![image](https://user-images.githubusercontent.com/15063294/101458923-d5ed1100-3937-11eb-8127-438841a2d733.png)

2. Crashes shortly after that if it finds tests. If it doesn't nothing happens but the icon continues

![image](https://user-images.githubusercontent.com/15063294/101458834-ab9b5380-3937-11eb-870f-c9173ec5b739.png)


```
localProcessExtensionHost.ts:257 Extension Host
localProcessExtensionHost.ts:258 [1208/092744.871:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
abstractExtensionService.ts:840 Extension 'ms-vscode.powershell-preview cannot use PROPOSED API (must started out of dev or enabled via --enable-proposed-api)
updateEnableProposedApi @ abstractExtensionService.ts:840
abstractExtensionService.ts:721 [ms-vscode.powershell-preview]: Command `workbench.action.debug.start` appears multiple times in the `commands` section.
abstractExtensionService.ts:721 [ms-vscode.powershell-preview]: Command `workbench.action.closePanel` appears multiple times in the `commands` section.
abstractExtensionService.ts:721 [ms-vscode.powershell-preview]: Command `workbench.action.positionPanelLeft` appears multiple times in the `commands` section.
abstractExtensionService.ts:721 [ms-vscode.powershell-preview]: Command `workbench.action.positionPanelBottom` appears multiple times in the `commands` section.
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope source.yaml.
Old grammar file: file:///c%3A/Users/przemyslaw.klys/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/extensions/yaml/syntaxes/yaml.tmLanguage.json.
New grammar file: file:///c%3A/Users/przemyslaw.klys/.vscode/extensions/redhat.vscode-yaml-0.13.0/syntaxes/yaml.tmLanguage.json
register @ TMScopeRegistry.ts:43
console.ts:137 [Extension Host] swdc-vscode: Loaded softwaredotcom.swdc-vscode v2.3.21
console.ts:137 [Extension Host] swdc-tracker initialized and ready to send events to com-software-prod1.collector.snplow.net
console.ts:137 [Extension Host] (node:18640) UnhandledPromiseRejectionWarning: [object Uint8Array]
t.log @ console.ts:140
console.ts:137 [Extension Host] (node:18640) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
t.log @ console.ts:140
console.ts:137 [Extension Host] (node:18640) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
t.log @ console.ts:140
abstractExtensionService.ts:467 Extension host terminated unexpectedly. Code:  1  Signal:  null
_onExtensionHostCrashed @ abstractExtensionService.ts:438
log.ts:197   ERR Extension host terminated unexpectedly. The following extensions were running: vscode.microsoft-authentication, vscode.powershell, vscode.debug-auto-launch, vscode.emmet, vscode.git, vscode.merge-conflict, vscode.search-result, 2gua.rainbow-brackets, alefragnani.project-manager, hbenl.vscode-test-explorer, PKief.material-icon-theme, pranaygp.vscode-css-peek, saekiraku.pin-up, steditor.line-endings, WakaTime.vscode-wakatime, yzhang.markdown-all-in-one, ms-vscode.powershell-preview, vscode.github, TylerLeonhardt.vscode-pester-test-adapter, ms-vscode.js-debug, eamodio.gitlens, GitHub.vscode-pull-request-github, softwaredotcom.swdc-vscode, usernamehw.errorlens, vscode.github-authentication
notificationsAlerts.ts:40 Extension host terminated unexpectedly.
onDidChangeNotification @ breadcrumbsModel.ts:39
localProcessExtensionHost.ts:257 Extension Host
localProcessExtensionHost.ts:258 undefined:3
ModuleName: PSTeams Version: 2.0.1
^

[SyntaxError: Unexpected token M in JSON at position 2
	at JSON.parse (<anonymous>)
	at ChildProcess.<anonymous> (c:\Users\przemyslaw.klys\.vscode\extensions\tylerleonhardt.vscode-pester-test-adapter-0.0.5\out\pesterTests.js:53:48)
	at ChildProcess.emit (events.js:223:5)
	at maybeClose (internal/child_process.js:1021:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)]
localProcessExtensionHost.ts:257 Extension Host
localProcessExtensionHost.ts:258 [1208/092820.906:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
log.ts:197   ERR No authentication provider 'microsoft' is currently registered.: Error: No authentication provider 'microsoft' is currently registered.
    at E.getSessions (file:///C:/Users/przemyslaw.klys/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:4345:744)
    at async e.getAccounts (file:///C:/Users/przemyslaw.klys/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6166:660)
    at async e.update (file:///C:/Users/przemyslaw.klys/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6166:397)
console.ts:137 [Extension Host] swdc-vscode: Loaded softwaredotcom.swdc-vscode v2.3.21
console.ts:137 [Extension Host] swdc-tracker initialized and ready to send events to com-software-prod1.collector.snplow.net
console.ts:137 [Extension Host] (node:34652) UnhandledPromiseRejectionWarning: [object Uint8Array]
t.log @ console.ts:140
console.ts:137 [Extension Host] (node:34652) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
t.log @ console.ts:140
console.ts:137 [Extension Host] (node:34652) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
t.log @ console.ts:140
abstractExtensionService.ts:467 Extension host terminated unexpectedly. Code:  1  Signal:  null
_onExtensionHostCrashed @ abstractExtensionService.ts:438
log.ts:197   ERR Extension host terminated unexpectedly. The following extensions were running: vscode.powershell, vscode.microsoft-authentication, ms-vscode.js-debug, vscode.debug-auto-launch, vscode.emmet, vscode.git, vscode.merge-conflict, vscode.search-result, 2gua.rainbow-brackets, alefragnani.project-manager, hbenl.vscode-test-explorer, PKief.material-icon-theme, pranaygp.vscode-css-peek, saekiraku.pin-up, steditor.line-endings, WakaTime.vscode-wakatime, vscode.github-authentication, eamodio.gitlens, vscode.debug-server-ready, yzhang.markdown-all-in-one, ms-vscode.powershell-preview, vscode.github, TylerLeonhardt.vscode-pester-test-adapter, GitHub.vscode-pull-request-github, softwaredotcom.swdc-vscode, usernamehw.errorlens
notificationsAlerts.ts:40 Extension host terminated unexpectedly.
onDidChangeNotification @ breadcrumbsModel.ts:39
localProcessExtensionHost.ts:257 Extension Host
localProcessExtensionHost.ts:258 undefined:3
ModuleName: PSTeams Version: 2.0.1
^

[SyntaxError: Unexpected token M in JSON at position 2
	at JSON.parse (<anonymous>)
	at ChildProcess.<anonymous> (c:\Users\przemyslaw.klys\.vscode\extensions\tylerleonhardt.vscode-pester-test-adapter-0.0.5\out\pesterTests.js:53:48)
	at ChildProcess.emit (events.js:223:5)
	at maybeClose (internal/child_process.js:1021:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)]
DevTools failed to load SourceMap: Could not load content for https://ticino.blob.core.windows.net/sourcemaps/e5a624b788d92b8d34d1392e4c4d9789406efe8f/node_modules/windows-process-tree/lib/index.js.map: Unknown error
```

I have opened it, and then let it crash and was able to get logs

```
[2020-12-11 08:51:27.703] [INFO] Test Explorer found
[2020-12-11 08:51:27.703] [INFO] Creating adapter for c:\Support\GitHub\GpoZaurr
[2020-12-11 08:51:27.703] [INFO] Initializing Pester adapter
[2020-12-11 08:51:27.703] [INFO] Initializing Pester test runner.
[2020-12-11 08:51:27.704] [INFO] Registering adapter for c:\Support\GitHub\GpoZaurr
[2020-12-11 08:51:27.705] [INFO] Loading Pester tests
[2020-12-11 08:51:27.705] [INFO] Initialization finished
[2020-12-11 08:51:27.785] [DEBUG] Found 3 paths
[2020-12-11 08:51:31.131] [DEBUG] Using Windows PowerShell (x64) at: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
[2020-12-11 08:51:31.414] [ERROR] stderr: Set-PSReadLineOption : The predictive suggestion feature cannot be enabled because the console output doesn't support v
irtual terminal processing or it's redirected.

[2020-12-11 08:51:31.415] [ERROR] stderr: At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:1 char:1
+ Set-PSReadLineOption -PredictionSource History -PredictionViewStyle L ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-PSReadLineOption], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.SetPSReadLineOption
 

[2020-12-11 08:51:33.175] [DEBUG] stdout: 

[2020-12-11 08:51:33.175] [DEBUG] stdout: 

[2020-12-11 08:51:33.182] [DEBUG] stdout: ModuleName: 
[2020-12-11 08:51:33.182] [DEBUG] stdout: GPOZaurr
[2020-12-11 08:51:33.183] [DEBUG] stdout:  Version: 
[2020-12-11 08:51:33.183] [DEBUG] stdout: 0.0.98
[2020-12-11 08:51:33.185] [DEBUG] stdout: 

[2020-12-11 08:51:33.186] [DEBUG] stdout: PowerShell Version: 
[2020-12-11 08:51:33.187] [DEBUG] stdout: 5.1.19041.610
[2020-12-11 08:51:33.187] [DEBUG] stdout: 

[2020-12-11 08:51:33.188] [DEBUG] stdout: PowerShell Edition: 
[2020-12-11 08:51:33.189] [DEBUG] stdout: Desktop
[2020-12-11 08:51:33.189] [DEBUG] stdout: 

[2020-12-11 08:51:33.190] [DEBUG] stdout: Required modules: 
[2020-12-11 08:51:33.190] [DEBUG] stdout: 

[2020-12-11 08:51:33.190] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.191] [DEBUG] stdout: PSSharedGoods
[2020-12-11 08:51:33.191] [DEBUG] stdout:  Version: 
[2020-12-11 08:51:33.191] [DEBUG] stdout: 0.0.190
[2020-12-11 08:51:33.191] [DEBUG] stdout: 

[2020-12-11 08:51:33.192] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.192] [DEBUG] stdout: ADEssentials
[2020-12-11 08:51:33.192] [DEBUG] stdout:  Version: 
[2020-12-11 08:51:33.193] [DEBUG] stdout: 0.0.103
[2020-12-11 08:51:33.193] [DEBUG] stdout: 

[2020-12-11 08:51:33.193] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.194] [DEBUG] stdout: PSWriteHTML
[2020-12-11 08:51:33.194] [DEBUG] stdout:  Version: 
[2020-12-11 08:51:33.194] [DEBUG] stdout: 0.0.122
[2020-12-11 08:51:33.195] [DEBUG] stdout: 

[2020-12-11 08:51:33.195] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.195] [DEBUG] stdout: CimCmdlets
[2020-12-11 08:51:33.196] [DEBUG] stdout: 

[2020-12-11 08:51:33.196] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.196] [DEBUG] stdout: Microsoft.PowerShell.Management
[2020-12-11 08:51:33.197] [DEBUG] stdout: 

[2020-12-11 08:51:33.197] [DEBUG] stdout:    [>] 
[2020-12-11 08:51:33.197] [DEBUG] stdout: Microsoft.PowerShell.Utility
[2020-12-11 08:51:33.198] [DEBUG] stdout: 

[2020-12-11 08:51:33.200] [DEBUG] stdout:    [>] Microsoft.PowerShell.Security


[2020-12-11 08:51:35.313] [DEBUG] stdout: 
Starting discovery in 2 files.
[2020-12-11 08:51:35.313] [DEBUG] stdout: 

[2020-12-11 08:51:35.366] [DEBUG] stdout: Discovering in GPOOwners.Tests.ps1.
[2020-12-11 08:51:35.366] [DEBUG] stdout: 

[2020-12-11 08:51:35.584] [DEBUG] stdout: Found 5 tests. 59ms

[2020-12-11 08:51:35.585] [DEBUG] stdout: Discovering in GPOPermissions.Tests.ps1.
[2020-12-11 08:51:35.585] [DEBUG] stdout: 

[2020-12-11 08:51:35.591] [DEBUG] stdout: Found 5 tests. 6ms

[2020-12-11 08:51:35.701] [DEBUG] stdout: Discovery finished in 392ms.
[2020-12-11 08:51:35.702] [DEBUG] stdout: 

[2020-12-11 08:51:35.801] [DEBUG] stdout: 
Running tests from 'GPOOwners.Tests.ps1'
[2020-12-11 08:51:35.801] [DEBUG] stdout: 

[2020-12-11 08:51:36.823] [DEBUG] stdout: Describing GPO Owners Management - Simple
[2020-12-11 08:51:36.823] [DEBUG] stdout: 

[2020-12-11 08:51:40.970] [DEBUG] stdout:   [+] Get-GPOZaurrOwner - Should return proper data
[2020-12-11 08:51:40.972] [DEBUG] stdout:  4.07s (3.92s|151ms)
[2020-12-11 08:51:40.972] [DEBUG] stdout: 

[2020-12-11 08:51:41.462] [DEBUG] stdout: VERBOSE: Get-GPOZaurrOwner - Processing GPO [1/]: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz
[2020-12-11 08:51:41.462] [DEBUG] stdout: 

[2020-12-11 08:51:41.494] [DEBUG] stdout: VERBOSE: Set-GPOZaurrOwner - Force was used to push new owner to TEST | GPOZaurr Permissions Testing from domain: 
ad.evotec.xyz - owner EVOTEC\Domain Admins / sysvol owner EVOTEC\Domain Admins.
[2020-12-11 08:51:41.495] [DEBUG] stdout: 

[2020-12-11 08:51:41.497] [DEBUG] stdout: VERBOSE: Set-GPOZaurrOwner - Changing GPO: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz from owner 
EVOTEC\Domain Admins (SID: S-1-5-21-853615985-2870445339-3163598659-512) to przemyslaw.klys
[2020-12-11 08:51:41.497] [DEBUG] stdout: 

[2020-12-11 08:51:41.541] [DEBUG] stdout: VERBOSE: Set-GPOZaurrOwner - Changing Sysvol Owner GPO: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz 
from owner EVOTEC\Domain Admins (SID: S-1-5-21-853615985-2870445339-3163598659-512) to przemyslaw.klys
[2020-12-11 08:51:41.541] [DEBUG] stdout: 

[2020-12-11 08:51:41.561] [DEBUG] stdout: VERBOSE: Performing the operation "Replacing owner EVOTEC\Domain Admins to przemyslaw.klys" on target 
"\\ad.evotec.xyz\SysVol\ad.evotec.xyz\Policies\{19EFFAE7-AB10-4854-90F9-44DEFDADE858}".
[2020-12-11 08:51:41.562] [DEBUG] stdout: 

[2020-12-11 08:51:41.652] [DEBUG] stdout:   [+] Set-GPOZaurrOwner - Should set proper data
[2020-12-11 08:51:41.652] [DEBUG] stdout:  678ms (677ms|1ms)

[2020-12-11 08:51:42.113] [DEBUG] stdout:   [+] Get-GPOZaurrOwner - Should return proper data for one GPO
[2020-12-11 08:51:42.113] [DEBUG] stdout:  459ms (458ms|1ms)
[2020-12-11 08:51:42.113] [DEBUG] stdout: 

[2020-12-11 08:51:42.485] [DEBUG] stdout: VERBOSE: Get-GPOZaurrOwner - Processing GPO [1/]: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz
[2020-12-11 08:51:42.486] [DEBUG] stdout: 

[2020-12-11 08:51:42.513] [DEBUG] stdout: VERBOSE: Set-GPOZaurrOwner - Changing GPO: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz from owner 
EVOTEC\przemyslaw.klys (SID: S-1-5-21-853615985-2870445339-3163598659-1105) to EVOTEC\Domain Admins
[2020-12-11 08:51:42.513] [DEBUG] stdout: 

[2020-12-11 08:51:42.538] [DEBUG] stdout: VERBOSE: Set-GPOZaurrOwner - Changing Sysvol Owner GPO: TEST | GPOZaurr Permissions Testing from domain: ad.evotec.xyz 
from owner EVOTEC\przemyslaw.klys (SID: S-1-5-21-853615985-2870445339-3163598659-1105) to EVOTEC\Domain Admins
[2020-12-11 08:51:42.538] [DEBUG] stdout: 

[2020-12-11 08:51:42.547] [DEBUG] stdout: VERBOSE: Performing the operation "Replacing owner EVOTEC\przemyslaw.klys to EVOTEC\Domain Admins" on target 
"\\ad.evotec.xyz\SysVol\ad.evotec.xyz\Policies\{19EFFAE7-AB10-4854-90F9-44DEFDADE858}".
[2020-12-11 08:51:42.547] [DEBUG] stdout: 

[2020-12-11 08:51:42.658] [DEBUG] stdout:   [+] Set-GPOZaurrOwner - Should set proper data
[2020-12-11 08:51:42.659] [DEBUG] stdout:  542ms (541ms|1ms)
[2020-12-11 08:51:42.659] [DEBUG] stdout: 

[2020-12-11 08:51:43.019] [DEBUG] stdout:   [+] Get-GPOZaurrOwner - Should return proper data for one GPO (Domain Admins)
[2020-12-11 08:51:43.019] [DEBUG] stdout:  358ms (357ms|1ms)
[2020-12-11 08:51:43.020] [DEBUG] stdout: 

[2020-12-11 08:51:43.477] [DEBUG] stdout: 
Running tests from 'GPOPermissions.Tests.ps1'
[2020-12-11 08:51:43.477] [DEBUG] stdout: 

[2020-12-11 08:51:44.378] [DEBUG] stdout: VERBOSE: Get-PrivPermission - Processing TEST | GPOZaurr Permissions Testing from ad.evotec.xyz
[2020-12-11 08:51:44.378] [DEBUG] stdout: 

[2020-12-11 08:51:44.432] [DEBUG] stdout: Describing GPO Permissions Management - Simple
[2020-12-11 08:51:44.432] [DEBUG] stdout: 

[2020-12-11 08:51:46.585] [DEBUG] stdout:   [+] Get-GPOZaurrPermission - Should return proper data
[2020-12-11 08:51:46.586] [DEBUG] stdout:  2.15s (2.15s|3ms)
[2020-12-11 08:51:46.586] [DEBUG] stdout: 

[2020-12-11 08:51:47.020] [DEBUG] stdout: VERBOSE: Get-PrivPermission - Processing TEST | GPOZaurr Permissions Testing from ad.evotec.xyz
[2020-12-11 08:51:47.020] [DEBUG] stdout: 

[2020-12-11 08:51:47.044] [DEBUG] stdout: VERBOSE: Add-GPOZaurrPermission - Adding permission GpoEdit for przemyslaw.klys to TEST | GPOZaurr Permissions Testing 
at ad.evotec.xyz
[2020-12-11 08:51:47.045] [DEBUG] stdout: 

[2020-12-11 08:51:47.046] [DEBUG] stdout: What if: Performing the operation "Adding przemyslaw.klys / GpoEdit to TEST | GPOZaurr Permissions Testing at ad.evotec.xyz" on target "TEST | GPOZaurr Permissions Testing".

[2020-12-11 08:51:47.290] [DEBUG] stdout:   [+] Add-GPOZaurrPermission - With WHATIF works
[2020-12-11 08:51:47.290] [DEBUG] stdout:  702ms (700ms|1ms)
[2020-12-11 08:51:47.290] [DEBUG] stdout: 

[2020-12-11 08:51:47.710] [DEBUG] stdout: VERBOSE: Get-PrivPermission - Processing TEST | GPOZaurr Permissions Testing from ad.evotec.xyz
[2020-12-11 08:51:47.710] [DEBUG] stdout: 

[2020-12-11 08:51:47.714] [DEBUG] stdout: VERBOSE: Add-GPOZaurrPermission - Adding permission GpoEdit for przemyslaw.klys to TEST | GPOZaurr Permissions Testing 
at ad.evotec.xyz
[2020-12-11 08:51:47.714] [DEBUG] stdout: 

[2020-12-11 08:51:47.715] [DEBUG] stdout: VERBOSE: Performing the operation "Adding przemyslaw.klys / GpoEdit to TEST | GPOZaurr Permissions Testing at 
ad.evotec.xyz" on target "TEST | GPOZaurr Permissions Testing".
[2020-12-11 08:51:47.715] [DEBUG] stdout: 

[2020-12-11 08:51:47.716] [DEBUG] stdout: VERBOSE: Add-GPOZaurrPermission - Adding permission GpoEdit for przemyslaw.klys

[2020-12-11 08:51:48.131] [DEBUG] stdout:   [+] Add-GPOZaurrPermission - Without WHATIF works
[2020-12-11 08:51:48.132] [DEBUG] stdout:  839ms (838ms|1ms)
[2020-12-11 08:51:48.132] [DEBUG] stdout: 

[2020-12-11 08:51:48.261] [DEBUG] stdout: VERBOSE: Get-PrivPermission - Processing TEST | GPOZaurr Permissions Testing from ad.evotec.xyz
[2020-12-11 08:51:48.262] [DEBUG] stdout: 

[2020-12-11 08:51:48.284] [DEBUG] stdout: What if: Performing the operation "Removing SID: S-1-5-21-853615985-2870445339-3163598659-1105, Name: ad.evotec.xyz\Przemysław Kłys, SidType: NotAdministrative from domain ad.evotec.xyz" on target "TEST | GPOZaurr Permissions Testing".

[2020-12-11 08:51:48.577] [DEBUG] stdout:   [+] Remove-GPOZaurrPermission - With WHATIF
[2020-12-11 08:51:48.577] [DEBUG] stdout:  444ms (443ms|1ms)
[2020-12-11 08:51:48.577] [DEBUG] stdout: 

[2020-12-11 08:51:48.799] [DEBUG] stdout: VERBOSE: Get-PrivPermission - Processing TEST | GPOZaurr Permissions Testing from ad.evotec.xyz
[2020-12-11 08:51:48.799] [DEBUG] stdout: 

[2020-12-11 08:51:48.809] [DEBUG] stdout: VERBOSE: Remove-GPOZaurrPermission - Removing permission GpoEdit for S-1-5-21-853615985-2870445339-3163598659-1105 / 
Przemysław Kłys / Type: Allow
[2020-12-11 08:51:48.809] [DEBUG] stdout: 

[2020-12-11 08:51:49.116] [DEBUG] stdout:   [+] Remove-GPOZaurrPermission - Without WHATIF
[2020-12-11 08:51:49.117] [DEBUG] stdout:  537ms (537ms|1ms)
[2020-12-11 08:51:49.117] [DEBUG] stdout: 

[2020-12-11 08:51:49.411] [DEBUG] stdout: Tests completed in 13.76s
[2020-12-11 08:51:49.411] [DEBUG] stdout: 

[2020-12-11 08:51:49.417] [DEBUG] stdout: Tests Passed: 10, 
[2020-12-11 08:51:49.418] [DEBUG] stdout: Failed: 0, 
[2020-12-11 08:51:49.420] [DEBUG] stdout: Skipped: 0 
[2020-12-11 08:51:49.422] [DEBUG] stdout: NotRun: 0
[2020-12-11 08:51:49.422] [DEBUG] stdout: 

[2020-12-11 08:51:49.691] [DEBUG] stdout: {
    "type":  "suite",
    "id":  "root",
    "label":  "Pester",
    "children":  [
                     {
                         "type":  "suite",
                         "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                         "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                         "label":  "GPOOwners.Tests.ps1",
                         "children":  [
                                          {
                                              "type":  "suite",
                                              "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;1",
                                              "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                              "line":  0,
                                              "label":  "GPO Owners Management - Simple",
                                              "children":  [
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;7",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                                                   "line":  6,
                                                                   "label":  "Get-GPOZaurrOwner - Should return proper data"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;18",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                                                   "line":  17,
                                                                   "label":  "Set-GPOZaurrOwner - Should set proper data"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;21",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                                                   "line":  20,
                                                                   "label":  "Get-GPOZaurrOwner - Should return proper data for one GPO"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;30",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                                                   "line":  29,
                                                                   "label":  "Set-GPOZaurrOwner - Should set proper data"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1;33",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOOwners.Tests.ps1",
                                                                   "line":  32,
                                                                   "label":  "Get-GPOZaurrOwner - Should return proper data for one GPO (Domain Admins)"
                                                               }
                                                           ]
                                          }
                                      ]
                     },
                     {
                         "type":  "suite",
                         "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                         "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                         "label":  "GPOPermissions.Tests.ps1",
                         "children":  [
                                          {
                                              "type":  "suite",
                                              "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;1",
                                              "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                              "line":  0,
                                              "label":  "GPO Permissions Management - Simple",
                                              "children":  [
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;7",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                                                   "line":  6,
                                                                   "label":  "Get-GPOZaurrPermission - Should return proper data"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;42",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                                                   "line":  41,
                                                                   "label":  "Add-GPOZaurrPermission - With WHATIF works"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;49",
                                          
[2020-12-11 08:51:49.691] [DEBUG] stdout:                          "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                                                   "line":  48,
                                                                   "label":  "Add-GPOZaurrPermission - Without WHATIF works"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;56",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                                                   "line":  55,
                                                                   "label":  "Remove-GPOZaurrPermission - With WHATIF"
                                                               },
                                                               {
                                                                   "type":  "test",
                                                                   "id":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1;65",
                                                                   "file":  "C:\\Support\\GitHub\\GpoZaurr\\Tests\\GPOPermissions.Tests.ps1",
                                                                   "line":  64,
                                                                   "label":  "Remove-GPOZaurrPermission - Without WHATIF"
                                                               }
                                                           ]
                                          }
                                      ]
                     },
                     {
                         "type":  "suite",
                         "id":  "C:\\Support\\GitHub\\GpoZaurr\\GPOZaurr.Tests.ps1",
                         "file":  "C:\\Support\\GitHub\\GpoZaurr\\GPOZaurr.Tests.ps1",
                         "label":  "GPOZaurr.Tests.ps1",
                         "children":  [

                                      ]
                     }
                 ]
}
```

Yes, doesn't crash now

```
[2020-12-11 19:47:59.660] [INFO] Test Explorer found
[2020-12-11 19:47:59.660] [INFO] Creating adapter for c:\Support\GitHub\GpoZaurr
[2020-12-11 19:47:59.661] [INFO] Initializing Pester adapter
[2020-12-11 19:47:59.661] [INFO] Initializing Pester test runner.
[2020-12-11 19:47:59.662] [INFO] Registering adapter for c:\Support\GitHub\GpoZaurr
[2020-12-11 19:47:59.662] [INFO] Loading Pester tests
[2020-12-11 19:47:59.662] [INFO] Initialization finished
[2020-12-11 19:47:59.817] [DEBUG] Found 3 paths
[2020-12-11 19:47:59.818] [DEBUG] Using Windows PowerShell (x64) at: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
[2020-12-11 19:48:00.781] [ERROR] stderr: Test-Path : Illegal characters in path.

[2020-12-11 19:48:00.781] [ERROR] stderr: At C:\Program Files\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:3313 char:18
+             if ((& $script:SafeCommands['Test-Path'] $p)) {
+                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (C:\Support\GitH...Zaurr.Tests.ps1:String) [Test-Path], ArgumentExcepti 
   on
    + FullyQualifiedErrorId : ItemExistsArgumentError,Microsoft.PowerShell.Commands.TestPathCommand
 

[2020-12-11 19:48:00.786] [ERROR] stderr: Get-ChildItem : Illegal characters in path.

[2020-12-11 19:48:00.786] [ERROR] stderr: At C:\Program Files\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:3339 char:13
+             & $SafeCommands['Get-ChildItem'] -Recurse -Path $p -Filte ...

[2020-12-11 19:48:00.786] [ERROR] stderr: +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-ChildItem], ArgumentException

[2020-12-11 19:48:00.786] [ERROR] stderr:     + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.GetChildItemCommand

[2020-12-11 19:48:00.786] [ERROR] stderr:  

[2020-12-11 19:48:00.867] [ERROR] stderr: Find-Test : Cannot bind argument to parameter 'BlockContainer' because it is null.

[2020-12-11 19:48:00.867] [ERROR] stderr: At line:30 char:35
+         Find-Test -BlockContainer $containers -SessionState $SessionS ...
+                                   ~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Find-Test], ParameterBindingValidationException

[2020-12-11 19:48:00.867] [ERROR] stderr:     + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Find-Test
 

[2020-12-11 19:48:00.877] [DEBUG] stdout: {
    "type":  "suite",
    "id":  "root",
    "label":  "Pester",
    "children":  [

                 ]
}

[2020-12-11 19:48:00.899] [DEBUG] child process exited with code 0
[2020-12-11 19:48:00.966] [DEBUG] No test files found.
[2020-12-11 19:48:08.749] [INFO] Loading Pester tests
[2020-12-11 19:48:08.802] [DEBUG] Found 3 paths
[2020-12-11 19:48:08.802] [DEBUG] Using Windows PowerShell (x64) at: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
[2020-12-11 19:48:09.696] [ERROR] stderr: Test-Path : Illegal characters in path.

[2020-12-11 19:48:09.697] [ERROR] stderr: At C:\Program Files\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:3313 char:18
+             if ((& $script:SafeCommands['Test-Path'] $p)) {
+                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (C:\Support\GitH...Zaurr.Tests.ps1:String) [Test-Path], ArgumentExcepti 
   on
    + FullyQualifiedErrorId : ItemExistsArgumentError,Microsoft.PowerShell.Commands.TestPathCommand
 

[2020-12-11 19:48:09.701] [ERROR] stderr: Get-ChildItem : Illegal characters in path.

[2020-12-11 19:48:09.701] [ERROR] stderr: At C:\Program Files\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:3339 char:13
+             & $SafeCommands['Get-ChildItem'] -Recurse -Path $p -Filte ...

[2020-12-11 19:48:09.701] [ERROR] stderr: +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

[2020-12-11 19:48:09.701] [ERROR] stderr:     + CategoryInfo          : NotSpecified: (:) [Get-ChildItem], ArgumentException

[2020-12-11 19:48:09.701] [ERROR] stderr:     + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.GetChildItemCommand

[2020-12-11 19:48:09.701] [ERROR] stderr:  

[2020-12-11 19:48:09.778] [ERROR] stderr: Find-Test : Cannot bind argument to parameter 'BlockContainer' because it is null.

[2020-12-11 19:48:09.778] [ERROR] stderr: At line:30 char:35
+         Find-Test -BlockContainer $containers -SessionState $SessionS ...
+                                   ~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Find-Test], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Find-Test
 

[2020-12-11 19:48:09.787] [DEBUG] stdout: {
    "type":  "suite",
    "id":  "root",
    "label":  "Pester",
    "children":  [

                 ]
}

[2020-12-11 19:48:09.805] [DEBUG] child process exited with code 0
[2020-12-11 19:48:09.865] [DEBUG] No test files found.

```

![image](https://user-images.githubusercontent.com/15063294/101948446-53dc4100-3bf2-11eb-8914-b7c8ac82f140.png)

Tests are there, tests explorer shows no tests.

--- a/src/powershellScripts.ts
+++ b/src/powershellScripts.ts
@@ -27,6 +27,10 @@ function Discover-Test
         $invokedViaInvokePester = $true
         $files = Find-File -Path $Path -ExcludePath $ExcludePath -Extension $PesterPreference.Run.TestExtension.Value
         $containers = foreach ($f in $files) {
+            <# HACK: We check to see if there is a single Describe block in the file so that we don't accidentally execute code that shouldn't need to be executed. #>
+            if (!(Select-String -Path $f -SimpleMatch 'Describe')) {
+                continue
+            }
             New-BlockContainerObject -File (Get-Item $f)
         }
         Find-Test -BlockContainer $containers -SessionState $SessionState } -Path $Path -ExcludePath $ExcludePath -SessionState $PSCmdlet.SessionState

